app/src/main/java/com/randomappsinc/studentpicker/choosing/ChoicesDisplayDialog.java

```diff
@@ -22,13 +23,15 @@
     private ChoosingSettings settings;
     private int listId;
     private MaterialDialog dialog;
+    private ChoicesDisplayViewHolder choicesDisplayViewHolder;
 
-    ChoicesDisplayDialog(Listener listener, Context context, int listId, ChoosingSettings settings) {
+    ChoicesDisplayDialog(Listener listener, Context context, int listId, ListInfo listInfo, ChoosingSettings settings) {
         this.listId = listId;
         this.settings = settings;
         dialog = new MaterialDialog.Builder(context)
                 // Placeholder because otherwise, the view doesn't exist
                 .title(R.string.name_chosen)
+                .customView(R.layout.name_choices_dialog, false)
```

Comment:
Instead of custom view, I think we can just use MaterialDialog's .adapter() API. The dialog will just provide the RecyclerView for us :)